config/gnu-warnings/no-developer-general

```diff
@@ -6,3 +6,4 @@
 # regarding the library settings string (H4libhdf4_settings). We'll turn
 # it off here but leave it on in the developer flags.
 -Wno-overlength-strings
+-Wno-format-nonliteral
```

Comment:
These are just noise. We have no plans to rework how the library works in a way that would make these go away.